config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
-  get 'home/index'
-  post 'home/login'
+  get 'github/callback'
   get 'post/list'
   get 'post/edit'
   post 'post/preview'
   post 'post/submit'
-  root 'home#index'
+  root 'post#list'
```

Comment:
This looks like another instance where it's entirely the fault of the PO (me) for not clarifying exactly what the purpose of the application was. Here it looks like we're grabbing all existing posts to be able to edit those. Is that right?

Reply:
Yes, I intended for that view to display a list of posts to edit as well as have the functionality to add a new post like through an add button. For MVP I'll probably change the index to the post/edit route

Reply:
That would be perfect.